cores/arduino/stm32/lock_resource.h

```diff
@@ -35,15 +30,134 @@ typedef enum {
 } LockResource_Status_t;
 
 /* Exported constants --------------------------------------------------------*/
-#define LOCK_RESOURCE_TIMEOUT   100U /* timeout in ms */
+#if defined(STM32WBxx) // || defined(STM32H7xx)
```

Comment:
remove comment 
`// || defined(STM32H7xx)`

Reply:
OK. originally I add it for future use in case of H7 dual core support introduction :wink:
Done